### PR TITLE
WD-10160 Server download

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Update new section "Get started with Ubuntu today"
- Update 24.04 link (not ready until 24.04 release)
- Djlint formatting

## QA

- Check https://ubuntu-com-13750.demos.haus/download/server/thank-you-s390x against [copy updates](https://docs.google.com/document/d/1GptXtMfbgOjOmqb8DZCCTRwogkiP_cM95U9Z0gFSt7g/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10160

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/download/server/thank-you-s390x.html [hunk 1]
```diff
@@ -1,52 +1,122 @@
 {% extends "download/_base_download.html" %}
 
 {% block title %}Thanks for downloading Ubuntu Server for IBM Z and LinuxONE{% endblock %}
```
Comment (akbarkz) [inline templates/download/server/thank-you-s390x.html:3]: title is different in the copy doc

File: templates/download/server/thank-you-s390x.html [hunk 2]
```diff
@@ -1,52 +1,122 @@
...
+
 {% block canonical_url %}https://ubuntu.com/download/server/s390x{% endblock %}
-{% block meta_copydoc %}https://docs.google.com/document/d/1GptXtMfbgOjOmqb8DZCCTRwogkiP_cM95U9Z0gFSt7g/edit{% endblock meta_copydoc %}
```
Comment (akbarkz) [inline templates/download/server/thank-you-s390x.html:6]: missing description

**Review by akbarkz — CHANGES_REQUESTED** [review]

**Comment by akbarkz** [comment]
@lyubomir-popov I think we need more spacing here between these two sections. Wdyt?

**Comment by lyubomir-popov** [comment]
the hr here is not fixed width: 
image
this should be a p-section, not p-strip shallow 
image
can we use the new equal heights module here pls 
image

File: templates/download/server/thank-you-s390x.html [hunk 3]
```diff
@@ -1,52 +1,123 @@
 {% extends "download/_base_download.html" %}
 
-{% block title %}Thanks for downloading Ubuntu Server for IBM Z and LinuxONE{% endblock %}
+{% block title %}Thank you for downloading Ubuntu for LinuxONE and z Systems{% endblock %}
+{% block description %}Ubuntu is an open source software operating system that runs from the desktop, to the cloud, to all your internet connected things.{% endblock %}
```
Comment (akbarkz) [inline templates/download/server/thank-you-s390x.html:4]: it's supposed to be called `meta_description`

File: templates/download/server/thank-you-s390x.html (hunk 2 above)
Comment (carkod) [reply templates/download/server/thank-you-s390x.html:7]: Updated. Strange that it was showing when I did it this way 
Screenshot 2024-04-17 at 13 25 04

**Comment by carkod** [comment]
@lyubomir-popov are you happy with this?

**Comment by akbarkz** [comment]
@juanruitina should we remove the "Please select the release you wish to download." sentence since we only have one option?

**Comment by juanruitina** [comment]
> @juanruitina should we remove the "Please select the release you wish to download." sentence since we only have one option? Screenshot 2024-04-18 at 18 51 23

I'm OK with that, yes

**Pull request merged**